docs/Rakefile

```diff
@@ -6,7 +6,12 @@ task :default => :gen
 BUILD_DIR          = "build"
 MARKDOWN_FILES     = Rake::FileList.new("source/_tmp/*.md")
 TOPIC_MAP          = "#{BUILD_DIR}/_topic_map.yml"
-ADOC_VARIABLES     = "source/variables.adoc"
+ADOC_VARIABLES     = FileList.new('source/variables.adoc',
+ 'source/contributing/variables.adoc',
+ 'source/getting-started/variables.adoc',
+ 'source/openshift/variables.adoc',
+ 'source/using/variables.adoc'
+)
```

Comment:
> You broke it down because you thought there would be name/vale clash for variables?

No. The problem is with integration of Middleman and Asciibinder (docs.openshift.org). The latter requires includes to be relative to the root of the docs whereas Middleman does it relative to the current directory. Originally we had some customisation in _config.rb_, to make Middleman change the include path. This worked ok. Then recently we made it possible to add images as well. There things are relative, in this case also in AsciiBinder. I actually removed at this stage the customisation. This made image includes work, but broke accidentally the includes of variables. 

We could try to get the customisations back, but then we would somehow have to differentiate between image includes and adoc includes. I think this would be hard. The alternative is to duplicate the variables file, so that it works in all cases. In the case of Middleman, it uses the variables.adoc in each directory. docs.openshift.org will only use the top level one. Since they are all the same, no harm is done. Seemed to be the easiest way to "fix" it.